### PR TITLE
https://issues.redhat.com/browse/ACM-14559 Proxy settings

### PR DESCRIPTION
**Pull request by oafischer**
2.12 only

### CODE REVIEW

File: clusters/cluster_lifecycle/adv_config_cluster.adoc [hunk 1]
```diff
@@ -144,17 +144,25 @@ spec:
 +
 [source,yaml]
 ----
-apiVersion: config.open-cluster-management.io/v1alpha1
```
Comment (elgnay) [inline clusters/cluster_lifecycle/adv_config_cluster.adoc]: We still need to keep this line. Apologies for the incorrect input in the draft content.

File: clusters/cluster_lifecycle/adv_config_cluster.adoc [hunk 2]
```diff
@@ -277,27 +293,31 @@ spec:
...
+In an environment that has a global `KlusterletConfig`, you can also create a cluster-specific `KlusterletConfig` and bind it with a managed cluster by adding the `agent.open-cluster-management.io/klusterlet-config: <klusterletconfig-name>` annotation to the `ManagedCluster resource`. The value of the cluster-specific `KlusterletConfig` overrides the global `KlusterletConfig` value if you set different values for the same field. 
 
-See the following example where the `hubKubeAPIServerURL` field has different values set in your `KlusterletConfig` and the global `KlusterletConfig`. The `"example.test.com"` value overrides the `"example.global.com"` value:
+See the following example where the `hubKubeAPIServerURL` field has different values set in your `KlusterletConfig` and the global `KlusterletConfig`. The "https://api.example.test.com:6443" value overrides the "https://api.example.global.com:6443" value:
 
 [source,yaml]
 ----
-apiVersion: config.open-cluster-management.io/v1alpha1
+piVersion: config.open-cluster-management.io/v1alpha1
```
Comment (elgnay) [inline clusters/cluster_lifecycle/adv_config_cluster.adoc]: It should be `apiVersion: config.open-cluster-management.io/v1alpha1`

File: clusters/cluster_lifecycle/adv_config_cluster.adoc [hunk 3]
```diff
@@ -277,27 +293,31 @@ spec:
...
   name: global
 spec:
-  hubKubeAPIServerURL: "example.global.com"
+  hubKubeAPIServerConfig:
+    url: "https://api.example.global.com:6443"
 ----
 
+The value of the global `KlusterletConfig` is used if there is no cluster-specific `KlusterletConfig` bound to a managed cluster, or the same field is missing or does not have a value in the cluster-specific `KlusterletConfig`. 
+
 If you create another `KlusterletConfig` in an environment that has a global `KlusterletConfig` and bind it with a managed cluster, the value of the global `KlusterletConfig` is used if the same field is missing or does not have a value in your `KlusterletConfig`.
```
Comment (elgnay) [inline clusters/cluster_lifecycle/adv_config_cluster.adoc]: This line can be removed as it has already been rephrased in the previous line.

**Comment by elgnay** [comment]
Those two lines (
https://github.com/stolostron/rhacm-docs/pull/7136/files#diff-7f135b7747ee558c9aa414118b8fde3a034098e226cbb24c0dcd6c6edc0e42f1L139-L140) should be updated as well.

**Comment by oafischer** [comment]
Made the requested updates, feel free to review again @elgnay. Thanks!

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**